diff --git a/dist/node/chunks/dep-689425f3.js b/dist/node/chunks/dep-689425f3.js
index babf8d5f63131b54ee2fca88a707db4ff3681448..019289dab7de9a08d1f94c3a6ad7fc90f6c0e3c7 100644
--- a/dist/node/chunks/dep-689425f3.js
+++ b/dist/node/chunks/dep-689425f3.js
@@ -38271,7 +38271,7 @@ const isModernFlag = `__VITE_IS_MODERN__`;
 const preloadMethod = `__vitePreload`;
 const preloadMarker = `__VITE_PRELOAD__`;
 const preloadBaseMarker = `__VITE_PRELOAD_BASE__`;
-const preloadHelperId = 'vite/preload-helper';
+const preloadHelperId = '\0vite/preload-helper';
 const preloadMarkerWithQuote = `"${preloadMarker}"`;
 const dynamicImportPrefixRE = /import\s*\(/;
 /**
