diff --git a/dist/node/chunks/dep-0a035c79.js b/dist/node/chunks/dep-0a035c79.js
index 57ca5f35e0ace9cacc9e61d4757ebc310fee4983..d39bfca26f674b1473e0c80a0894dc0bd4fb3cb8 100644
--- a/dist/node/chunks/dep-0a035c79.js
+++ b/dist/node/chunks/dep-0a035c79.js
@@ -38271,7 +38271,7 @@ const isModernFlag = `__VITE_IS_MODERN__`;
 const preloadMethod = `__vitePreload`;
 const preloadMarker = `__VITE_PRELOAD__`;
 const preloadBaseMarker = `__VITE_PRELOAD_BASE__`;
-const preloadHelperId = 'vite/preload-helper';
+const preloadHelperId = '\0vite/preload-helper';
 const preloadMarkerWithQuote = `"${preloadMarker}"`;
 const dynamicImportPrefixRE = /import\s*\(/;
 /**
